src/test/java/nextstep/ladder/LadderGameTest.java

```diff
@@ -52,4 +50,30 @@ void makeLadderLadderGameTest() {
             Assertions.assertThat(points).hasSize(testNames.size() - 1);
         }
     }
+
+    @Test
+    @DisplayName("사다리 게임 결과")
+    void makeLadderResultTest() {
+        //given
+        List<String> testNames = new ArrayList<>(List.of("pobi", "honux", "crong", "jk"));
+        int testHeight = 5;
+        MockLineGenerator lineGenerator = new MockLineGenerator();
+
+        LadderGame ladderGame = new LadderGame(testNames);
+
+        Ladder ladder = ladderGame.makeLadder(testHeight, lineGenerator);
+
+        List<Prize> prizes = List.of(new Prize("꽝"), new Prize("5000"), new Prize("꽝"), new Prize("3000"));
+
+        //when
+        LadderResult ladderResult = ladderGame.makeLadderResult(prizes);
+
+        //then
+        LinkedHashMap<Player, Prize> result = ladderResult.getAllResult();
+        Assertions.assertThat(result).hasSize(4);
+        Assertions.assertThat(result.get(new Player("pobi"))).isEqualTo(prizes.get(1));
```

Comment:
직관적으로 이해하기 쉽게  수 있게 검증하면 어떨까요?
```suggestion
        assertThat(result.get(new Player("pobi"))).isEqualTo(new Prize("5000"));
```